hooks/post_gen_project.py

```diff
@@ -1,24 +1,40 @@
 
-def gitlab_ci_enabled(adapter):
-    if adapter == "snowflake":
-        print(
-            "Set the following environment variables in your GitLab project to run dbt as part of your CI pipeline:\n"
-            " - DBT_ACCOUNT: Your snowflake account\n"
-            " - DBT_USER: Your snowflake username\n"
-            " - DBT_PASSWORD: Your snowflake password\n"
-            " - TEST_DATABASE: The Database name you use for running CI jobs\n\n"
-            "Learn more about GitLab CI/CD variables here: https://docs.gitlab.com/ee/ci/variables/"
-        )
+def gitlab_ci_enabled(adapter, ci_tool):
 
+  if ci_tool == "GitHub":
+    docs = "https://docs.github.com/en/actions/reference/environment-variables"
+  elif ci_tool == "GitLab":
+    docs = "https://docs.gitlab.com/ee/ci/variables/"
+
+  print(
+    "\nProject initialized successfully!\n"
+  )
+
+  if ci_tool == "None":
+    message = (
+      f"A base profile.yml file has been created in folder .config\n"
+      f"Edit it and move it to ~/.dbt\n"
```

Comment:
If we suggest moving that file, we break the CI pipeline. Instead, we should suggest copying it and changing credentials in the target `/.dbt/profiles.yml` file.